src/components/molecules/StartTable/StartTable.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useAsyncFn } from "react-use";
+
+import { updateVenueTable } from "api/table";
+
+import { Table } from "types/Table";
+import { VenueTemplate } from "types/venues";
+
+import { currentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import "./StartTable.scss";
+
+export interface StartTablePropsType {
+  tables: Table[];
+  newTable: Table;
+}
+
+export const StartTable: React.FC<StartTablePropsType> = ({
+  tables,
+  newTable,
+}) => {
+  const venue = useSelector(currentVenueSelector);
+
+  const [table] = tables;
+  const startTableWidth =
```

Comment:
styles hack for JazzBar template (tables components are aligned full width there)